pkg/statistics/analyze.go

```diff
@@ -61,6 +61,7 @@ func (h *AnalyzeTableID) Equals(t *AnalyzeTableID) bool {
 }
 
 // AnalyzeResult is used to represent analyze result.
+// AnalyzeResult represents the analyze result of one column/index.
```

Comment:
The subject is the same as the previous one, and it can be replaced with the pronoun "it".